client/src/app/shared/shared-main/misc/top-menu-dropdown.component.html

```diff
@@ -4,26 +4,28 @@
     <a *ngIf="menuEntry.routerLink" [routerLink]="menuEntry.routerLink" routerLinkActive="active" class="title-page title-page-settings">{{ menuEntry.label }}</a>
 
     <div *ngIf="!menuEntry.routerLink" ngbDropdown class="parent-entry"
-      #dropdown="ngbDropdown" (mouseleave)="closeDropdownIfHovered(dropdown)">
```

Comment:
Why did you remove the auto open on hover?

Reply:
It's in the description : 

> Improve Dropdown : active class on active link + remove hover open + close outside click only
make sure the sub-menu is not too long horizontally - less confusing nav and more common

Reply:
having both effects on click and hover is very confusing plus on hover effects, it still reminds messy navigation like https://ants.gouv.fr/ (it's not like I said the user sub-menu is messy, no, just globally the hover effect must be used with care, and thinking about our usage changes)